crates/iroha_executor_derive/src/entrypoint.rs

```diff
@@ -86,23 +85,16 @@ fn impl_validate_entrypoint(
         "Executor `{user_entrypoint_name}` entrypoint must have `Result` return type"
     );
 
-    block.stmts.insert(
-        0,
-        parse_quote!(
-            use ::iroha_executor::smart_contract::{ExecuteOnHost as _};
-        ),
-    );
```

Comment:
this was confusing. You would get to use `expect_dbg` inside the `main` function but if you had created another function then `expect_dbg` would require an import if called from there